doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -69,9 +69,9 @@ Joe is a Silver cardholder over the age of 25. He has had more than two previous
 
 * *Dispute Risk Rating*
 +
-Because Joe is a Silver cardholder and the disputed amount is 30 dollars, Joe has a rating of 1 on the *Dispute Risk Rating* table. If the disputed amount was 130 dollars, Joe would have a risk rating of 3.
+Because Joe is a Silver cardholder and the disputed amount is 40 dollars, Joe has a rating of 1 on the *Dispute Risk Rating* table. If the disputed amount was 140 dollars, Joe would have a risk rating of 3.
```

Comment:
Previous mentions of money were written as $44.50 and $4.50. Please change all monetary values to be the same.

Reply:
OK. The reason I used the the $ before is I was directly referring to his bill.